### ISSUE TITLE
Shipping form not displayed in checkout when user logged in

### ISSUE THREAD

**Issue by jibrandlr** [magento/magento2#6082]
### Preconditions

Magento 2.1.0, php 7.0.8, ubuntu 16.04 
### Steps to reproduce
1. Log in as a registered user.
2. Add an item to cart.
3. Go to checkout
### Expected result
1. Shipping form displayed in checkout.
### Actual result
1. Shipping form **not** displayed.

The shipping form is displayed correctly only when **guest user** reaches the checkout page.

![checkout-guest](https://cloud.githubusercontent.com/assets/8398744/17527243/2d6d222c-5e30-11e6-8fac-e10cdb034d4f.jpg)
![checkout-signedin](https://cloud.githubusercontent.com/assets/8398744/17527244/2d7223f8-5e30-11e6-86ee-8e104eaff9ad.jpg)

**Comment by zolthan** [comment]
Same issue here!

**Comment by AVoskoboinikov** [comment]
Hi, guys

Thanks for reporting this issue, but unfortunately I cannot reproduce it on our side using clean version of CE 2.1.0. I have shipping form for both type of users.

Please, try to clear all caches and reproduce it on your side. If you will succeed - give us your detailed Magento configuration (your custom settings; or maybe you have tried to modify code; external modules, etc.) and detailed steps to reproduce it. This will help us a lot.

Looking forward to getting response from you.

**Comment by lgarridoj** [comment]
@AzVo I deleted all caches and result is the same. I also removed the only extension (a payment extension) I have installed, no luck. Haven't made any modifications other than my theme customizations.
I compared the "Console" log in both situations (see image) bu there is no apparent error.
![console-checkout](https://cloud.githubusercontent.com/assets/8398744/17906490/30157d5e-693d-11e6-85fb-dfeea160df42.jpg)
